test/client/fixtures/sandbox/selection-test.js

```diff
@@ -0,0 +1,51 @@
+var selectionSandbox = hammerhead.eventSandbox.selection;
+var browserUtils     = hammerhead.utils.browser;
+
+var inputTestValue  = 'test@host.net';
+var isIE            = browserUtils.isIE;
+var isMobileBrowser = browserUtils.isIOS || browserUtils.isAndroid;
+
+var createTestInput = function () {
+    var input = $('<input type="email">').appendTo(document.body)[0];
```

Comment:
what? jquery is just a wrapper around native js. what's the problem to do the same as jquery?